web/v1/snowflake/sql-runner/sql/standard/02-page-views/01-main/07-commit-page-views.sql

```diff
@@ -0,0 +1,64 @@
+/*
+   Copyright 2021 Snowplow Analytics Ltd. All rights reserved.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+
+-- staged tables
+{{if eq .stage_next true}}
+
+  CALL {{.output_schema}}.commit_staged(
+    'FALSE',                                                       -- automigrate
+    UPPER('{{.scratch_schema}}.page_views_this_run{{.entropy}}'),  -- stage source
+    UPPER('{{.scratch_schema}}.page_views_staged{{.entropy}}'),    -- stage target
+    UPPER('page_view_id')                                          -- joinKey
+  );
+
+{{end}}
+
+-- production tables
+{{if ne (or .skip_derived false) true}}
+
+  CALL {{.output_schema}}.mk_transaction(
```

Comment:
How come we don't use the procedure here?

Reply:
We use the `mk_transaction` one, because this is the only thing that happens there. Trying to wrap the one in terms of the other just to end up to the one we already have felt like the complexity that was before.

Reply:
Ok cool. This approach is as valid as the other, so seems good to me. :)